.github/workflows/wheels.yml

```diff
@@ -31,35 +36,36 @@ jobs:
           python-version: '3.9'
 
       - uses: docker/setup-qemu-action@v1
-        if:  startsWith(matrix.os, 'ubuntu')
+        if: runner.os == 'Linux'
         name: Set up QEMU
 
       - name: Install prerequisites for macOS
-        if: contains(matrix.os, 'macos')
+        if: runner.os == 'macOS'
         env:
           # Best compatibility, even with older releases of macOS.
           MACOSX_DEPLOYMENT_TARGET: "10.9"
         run: |
           brew install --build-from-source --no-binaries --force szip
```

Comment:
Please do not use szip. I'm pretty sure that the license has some restrictions that are quite complex to handle.

Reply:
It's not needed anyway - I've been copying this part from #872 - which is why this is there in the first place.